src/features/offer/helpers/useOfferSummaryInfoList/useOfferSummaryInfoList.native.test.tsx

```diff
@@ -58,6 +59,28 @@ describe('useOfferSummaryInfoList', () => {
     mockdate.reset()
   })
 
+  it('should not return summaryInfoItems dates when offer stock has date in future and is cinema experience is available', async () => {
```

Comment:
`should not return summaryInfoItems dates when offer stock has date in future and is cinema offer`